model/src/w3metamd.F90

```diff
@@ -1,3 +1,20 @@
+!> @file w3metamd.F90
+!> @brief Dynamic storage for meta data attribute/value pairs.
+!> @author Chris Bunney @date 16-Dec-2020
+!/ ------------------------------------------------------------------- /
+!/
```

Comment:
In all the NCEPLIBS doxygen, we removed all the old-school text formatting that were in the original comments. So we removed things like:

```
!/ ------------------------------------------------------------------- /
!/
```

These are just extra work for everyone, and add nothing to the file.

Reply:
While some may believe it adds nothing, I'm sure there are some that would argue that it does add something:  visual structure.  I've decided to take an approach of not removing the existing legacy headers (including this type of formatting).  This way nothing is lost, and if we want to remove the headers and/or formatting in the future, this can easily be done in very little time.